### ISSUE TITLE
Blank white screen

### ISSUE THREAD

**Issue by mkdub69** [iam4x/pokemongo-webspoof#74]
Hi guys, I hope I don't annoy with noob questions of someone who does not know what he is doing. 

1. When the app starts on my phone, the screen is just blank white. Is this intentional? Do I have to switch back to the Pokemon Go App?

2. When running the App, nothing on my mac happens. There is no window showing the map and the arrows and all. Did I forget to do something?

Love and thanks in advance,

M

**Comment by AbelIngrand** [comment]
Hi, the blank app is normal, it's just a blank Xcode project that runs along side Pokemon Go to simulate the location. Once the blank app launched from Xcode, you can switch to Pokemon Go (just don't quit the blank app)

Electron (the framework used for the UI) can be sometimes really slow on old macs, wait a little bit or try relaunching the app.

**Comment by mkdub69** [comment]
Thanks for the help!

I have tried it on my Mid 2015 MBP, also relaunching several times. Do I need to get this Electron framework somewhere?

**Comment by AbelIngrand** [comment]
No, it's bundled with the app so there is nothing to install. Try reinstalling maybe. Seems strange that it takes so long on a 2015 MBP

**Comment by mkdub69** [comment]
Hm, my room mate who claims to know a little about xcode says that Electron does not start at all. Can you make something out of this?

**Comment by AbelIngrand** [comment]
Do you see and icon like this one ? ![Icon](https://upload.wikimedia.org/wikipedia/commons/3/3f/Electron_0.36.4_Icon.png) when launching the app or is there just Xcode launching ?

**Comment by mkdub69** [comment]
I did not see that icon. Nothing else launches.

**Comment by AbelIngrand** [comment]
Strange.. Try downloading latest version from here: https://github.com/iam4x/pokemongo-webspoof/releases/download/v1.1.0/pokemongo-webspoof-v110.app.tar.gz

**Comment by muescha** [comment]
better: add some instructions with the next steps todo instead of a complete white screen

**Comment by AbelIngrand** [comment]
Working on wedding a message on the blank screen

**Comment by masiarb** [comment]
I have the same issue. Electron is not starting despite the app launching fine on my device (blank screen). XCode just runs and no Electron GUI is showing on my Mac whatsoever (MBP retina mid 2012, 16GB RAM 2.3 GHz i7).

**Comment by adster93** [comment]
I can attest to this, seems to work on 1.0.2 but on that version I am having trouble still not getting pokemon.gpx on the debug menu.

**Comment by mkdub69** [comment]
Hi guys, downloading the version from Abel's link did the trick for me. Thanks for all the help!

**Comment by iam4x** [comment]
Thank's @AbelIngrand for the new screen that replace the blank iOS app screen, now you can click to open pokemongo directly 👍 

Anyway, app not starting is weird to help me debug:

1. Open Console.app (Applications > Utilities)
2. Enter `Electron` as search filter
3. Start `pokemongo-webpsoof.app` on Mac

Normally you will get additional log that can help me.
